src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4049,7 +4096,7 @@ private AccessorDeclarationSyntax ParseAccessorDeclaration(bool isEvent)
 
             var accMods = _pool.Allocate();
 
-            var accAttrs = this.ParseAttributeDeclarations();
+            var accAttrs = this.ParseAttributeDeclarations(inExpressionContext: true);
```

Comment:
>inExpressionContext: true [](http://example.com/codeflow?start=59&length=25)

Should this be `inExpressionContext: false`?

Reply:
Ack.  yes.